components/micro-gateway-cli/src/main/resources/templates/ballerinaToml.mustache

```diff
@@ -1,6 +1,38 @@
-{{#each this}}
-    [[platform.libraries]]
-        path = "$MGW-PROJECT_HOME/lib/{{this}}"
+[project]
+org-name= "wso2"
+version= "3.1.0"
 
+[dependencies]
+"wso2/gateway" = { path = "{{toolkitHome}}/lib/gateway-balo/gateway-2019r3-java8-3.1.0.balo"}
+{{#each dependencies}}
+{{{.}}}
 {{/each}}
 
+[platform]
+target = "java8"
+
+    [[platform.libraries]]
+        module = "jms"
+        path = "{{toolkitHome}}/lib/dependencies/andes-client-3.3.9.jar"
+        artifactId = "andes-client"
+        version = "3.3.9"
+        groupId = "org.wso2.andes.wso2"
```

Comment:
artifact id , group id are not supported. Hence shall we remove them

Reply:
what are the other unsupported attrs? how about `version`, `module`? Can we remove those too?

Reply:
I think we can keep only module and path, and remove the rest. Module gives an idea from which module this dependency comes